packages/nextjs/components/scaffold-eth/RainbowKitCustomConnectButton.tsx

```diff
@@ -6,7 +6,7 @@ import { getTargetNetwork } from "~~/utils/scaffold-eth";
 
 // todo: move this later scaffold config.  See TAutoConnect for comments on each prop
 const tempAutoConnectConfig: TAutoConnect = {
-  enableBurnerWallet: true,
+  enableBurnerWalletOnLoad: true,
```

Comment:
Should we move these options to `scaffold.config` ? 

It s only being used by `useAutoConnect` so maybe inline them at the top of  hook ? 

Confused a bit since I think people would like toggle this option